spec/booking_system_spec.rb

```diff
@@ -0,0 +1,242 @@
+require_relative 'spec_helper'
+require 'date'
+require 'pry'
+
+describe "BookingSystem class" do
+
+  before do
+    @system = Hotel::BookingSystem.new
+  end
+
+  describe "load rooms" do
+    it "loads rooms in an array" do
+      @rooms = @system.load_rooms
+      # binding.pry
+      expect(@rooms).must_be_kind_of Array
+    end
+  end
+
+  describe "make reservation" do
+    it "makes a reservation" do
+      @system.make_reservation(Date.new(2018,1,1), Date.new(2018,1,5))
+      reservation = Hotel::Reservation.new(reservation_id: 1, room: 1, start_date: Date.new(2018, 1, 1), end_date: Date.new(2018, 1, 5), price_per_night: 200)
+      expect(@system.reservations.length).must_equal 1
```

Comment:
You have a lot of `Date.new(...)` in this file, which is a little difficult to read. Instead you might create a couple in a before block and put them in instance variables to use later:

```ruby
@start_date = Date.new(2018, 1, 1)
@end_date = Date.new(2018, 1, 5)
```